pages/login.vue

```diff
@@ -86,11 +91,15 @@ export default {
   justify-content: center;
 }
 .v-form {
-  padding: 10px !important;
+  padding: 18px !important;
+  letter-spacing: 3px;
 }
 .v-card {
   margin: auto;
-  margin-top: 20px;
+  margin-top: 8%;
   max-width: 600px !important;
 }
+.v-text-field {
+  padding-top: 24px;
```

Comment:
same here, try to use `rem` or `em` if possible

Reply:
- replaced px with rem